### DISCUSSION TITLE
USB "rx_lanes: No such file or directory"

### DISCUSSION THREAD

**Discussion by MichaelPres** [OE4T/meta-tegra#1398] [Q&A] [answered]
Hi,
I got the following error message when running "lsusb -t". I am using branch "kirkstone-l4t-r32.7.x" with a TX2i on devkit. Do you have any clue on what is going on? 
```
 lsusb -t
/sys/bus/usb/devices/1-2.1/rx_lanes: No such file or directory
/sys/bus/usb/devices/1-2.1/tx_lanes: No such file or directory
/sys/bus/usb/devices/usb1/rx_lanes: No such file or directory
/sys/bus/usb/devices/usb1/tx_lanes: No such file or directory
/sys/bus/usb/devices/usb2/rx_lanes: No such file or directory
/sys/bus/usb/devices/usb2/tx_lanes: No such file or directory
/sys/bus/usb/devices/1-2.1.1/rx_lanes: No such file or directory
/sys/bus/usb/devices/1-2.1.1/tx_lanes: No such file or directory
/sys/bus/usb/devices/1-2/rx_lanes: No such file or directory
/sys/bus/usb/devices/1-2/tx_lanes: No such file or directory
/:  Bus 02.Port 1: Dev 1, Class=root_hub, Driver=tegra-xusb/3p, 5000M
/:  Bus 01.Port 1: Dev 1, Class=root_hub, Driver=tegra-xusb/4p, 480M
    |__ Port 2: Dev 2, If 0, Class=Hub, Driver=hub/4p, 480M
        |__ Port 1: Dev 3, If 0, Class=Hub, Driver=hub/4p, 480M
            |__ Port 1: Dev 4, If 0, Class=Vendor Specific Class, Driver=ftdi_sio, 12M
```
The more device I plug, the more error message I've got!

**Comment by madisongh** [comment, marked as answer]
It's because we have such an old kernel.  The support for the sysfs entries mentioned in those error messages were added in (from what I can tell) the 4.18 kernel, but we're still at 4.9.

It looks like the change to usbutils that expects the presence of those entries was made in version 014.

So either downgrading usbutils to version 013, or adding a patch to ignore the non-existence of the rx/tx lanes sysfs entries, should take care of the issue.

**Reply by MichaelPres** [reply]
Thank you very much Matt
